ibis/expr/types/structs.py

```diff
@@ -146,6 +146,15 @@ def __dir__(self):
     def _ipython_key_completions_(self) -> list[str]:
         return sorted(self.type().names)
 
+    def keys(self):
+        return self.names
```

Comment:
If this looks good I can add docstrings

Reply:
Seems fine.

Eventually we may want to cache the `values` and `items` results upon first use. Since expressions are immutable there's no reason to do those computations more than once. That be done later if it ever becomes an issue.